File: frontend/src/types/po/ProblemCase.ts
import type { Difficulty } from "../enum/Difficulty";
import type { ProblemState } from "../enum/ProblemStatus";

export default interface ProblemCase {
    id?: number;
    problemId: number;
    serial?: number;
    input: string;
    output: string;
    maxTimeMillis: number;
    maxMemory: number;
    createTime?: Date
}

File: backend/commons/src/main/java/top/cary61/carycode/commons/entity/po/ProblemCase.java
package top.cary61.carycode.commons.entity.po;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@TableName("problem_case")
public class ProblemCase {

    private Long id;

    private Long problemId;

    private Integer serial;

    private String input;

    private String output;

    private Integer maxTimeMillis;

    private Integer maxMemory;

    private LocalDateTime createTime;
}
